### PR TITLE
fresh install docs

### PR DESCRIPTION
**Pull request by ktzrbn**
minor corrections for readability, and also: 

switches locale from und-x-icu to C.UTF-8

und-x-icu will always through an error in this environment saying "Error: invalid locale name: "und-x-icu"

I am not sure if that's because vagrant box defaults to the location downloaded (for me, en_US.utf8) but regardless, C.UTF8 will cover everything and und-x-icu has not worked on any system I've tried

### CODE REVIEW

File: content/en/docs/Getting started/Installation/singleserverfreshinstall.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ For testing FOLIO installation on a PC, it's recommended to use Vagrant to separ
 
 2. Install a virtualization product.
 
-   For Windows, [install VirtualBox](https://www.virtualbox.org/).
+   For Windows, [use VirtualBox](https://www.virtualbox.org/).
 
 3. Install an Ubuntu box.
 
-   Create a `Vagrantfile` like the following.
+   Create or edit a `Vagrantfile` like the following in the same directory you created during the install process.
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverfreshinstall.md:56]: Hi Brandon,
there are many directories that were created during the install process, therefore this instruction is ambiguous.
The Vagrantfile can be in any directory and is not restricted to an existing directory. The user may create a new directory for the Vagrantfile.
Can you explain what problem this addition to the install instructions solves?

File: content/en/docs/Getting started/Installation/singleserverfreshinstall.md [hunk 2]
```diff
@@ -304,6 +304,8 @@ Once you have installed the requirements for Okapi and created a database, you c
       -d '{ "urls": [ "https://folio-registry.dev.folio.org" ] }' \
      http://localhost:9130/_/proxy/pull/modules
    ```
+   Be sure to replace "localhost" with your IP address. In Vagrant, this is usually 10.0.2.15.
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverfreshinstall.md:307]: Hi Brandon,
multiple people have succeeded with localhost.
Can you give more details in which configuration (operating system, etc.) localhost fails?

Comment (julianladisch) [reply content/en/docs/Getting started/Installation/singleserverfreshinstall.md:307]: And can you paste the complete error message?

**Review by inkuss — APPROVED** [review]

File: content/en/docs/Getting started/Installation/singleserverfreshinstall.md (hunk 2 above)
Comment (ktzrbn) [reply content/en/docs/Getting started/Installation/singleserverfreshinstall.md:307]: This just repeats the same directions from a previous line for consistency

File: content/en/docs/Getting started/Installation/singleserverfreshinstall.md (hunk 1 above)
Comment (ktzrbn) [reply content/en/docs/Getting started/Installation/singleserverfreshinstall.md:56]: I had originally written that users should create a project directory (based on Vagrant's documentation), but I removed that step--so if there's no project directory being created, no need for this line

**Pull request merged**